pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -46,7 +45,7 @@ type preFilterState struct {
 	// it's not guaranteed to be the 2nd minimum match number.
 	TpKeyToCriticalPaths map[string]*criticalPaths
 	// TpPairToMatchNum is keyed with topologyPair, and valued with the number of matching pods.
-	TpPairToMatchNum map[topologyPair]*int32
+	TpPairToMatchNum map[topologyPair]int32
```

Comment:
now that we are not using `atomic`, this can probably be just `map[topologyPair]int`

Reply:
It already is :)

Reply:
`int` instead of `int32`... but just a nit

Reply:
updated